youtube_dl/extractor/spankwire.py

```diff
@@ -54,62 +55,35 @@ def _real_extract(self, url):
         req = sanitized_Request('http://www.' + mobj.group('url'))
         req.add_header('Cookie', 'age_verified=1')
         webpage = self._download_webpage(req, video_id)
+        video_data = self._download_json(sanitized_Request('https://www.spankwire.com/api/video/' + video_id + '.json'), video_id)
 
-        title = self._html_search_regex(
-            r'<h1>([^<]+)', webpage, 'title')
-        description = self._html_search_regex(
-            r'(?s)<div\s+id="descriptionContent">(.+?)</div>',
-            webpage, 'description', fatal=False)
-        thumbnail = self._html_search_regex(
-            r'playerData\.screenShot\s*=\s*["\']([^"\']+)["\']',
-            webpage, 'thumbnail', fatal=False)
+        title = video_data.get('title')
+        description = video_data.get('description')
+        thumbnail = video_data.get('poster')
 
-        uploader = self._html_search_regex(
-            r'by:\s*<a [^>]*>(.+?)</a>',
-            webpage, 'uploader', fatal=False)
+        uploader = self._search_regex(
+            r'<a[^>]+class="uploaded__by"[^>]*>(.+?)</a>',
+            webpage, 'uploader', flags=re.DOTALL, fatal=False)
         uploader_id = self._html_search_regex(
-            r'by:\s*<a href="/(?:user/viewProfile|Profile\.aspx)\?.*?UserId=(\d+).*?"',
+            r'by\s*<a href="/(?:user/viewProfile|Profile\.aspx)\?.*?UserId=(\d+).*?"',
             webpage, 'uploader id', fatal=False)
         upload_date = unified_strdate(self._html_search_regex(
-            r'</a> on (.+?) at \d+:\d+',
+            r'</span>(.+?) at \d+:\d+ (AM|PM) by',
```

Comment:
Don't capture things you don't use.

Reply:
i have captured this as it already being done previously. Let me know if have to remove this itself.

Reply:
I'm not talking about capturing upload date. Do not capture AMPM.